v2/cmd/zlint/main.go

```diff
@@ -156,6 +161,10 @@ func doLint(inputFile *os.File, inform string, registry lint.Registry) {
 			log.Fatalf("can't format output: %s", err)
 		}
 		os.Stdout.Write(out.Bytes())
+	} else if summary {
+		formattedoutput.OutputSummary(zlintResult, false)
+	} else if longSummary {
+		formattedoutput.OutputSummary(zlintResult, true)
```

Comment:
tiny nit: It looks like providing both `-summary` and `-longSummary` will short-circuit to only displaying the short summary. I think it would be a little bit nicer to either error if both flags are enabled or to flip these two `else if` cases so that the long summary takes precedent if both are provided.

Reply:
I'm conflicted by this.  

Why do you think `longSummary` should take precedence?  My thinking was that shorter output is better, as as not to overwhelm the person running it.

Now that I'm looking at that whole if/else if/else block, I'm not too fond of it.  The original one was `prettyPrint` else `dump the json`.  It makes sense that those are exclusive, but it makes less sense to me that they are all exclusive.  I can imagine someone wanting all 3 of json output (pretty or not, but not both), a short summary, and a long summary.  That argues for pulling `summary` and `longSummary` out of that block and putting each of them in their own `if` statement.

Thoughts?

Reply:
> I'm conflicted by this.

That's fair, I don't think it's meaningful enough to derail forward progress. We can ignore this for now.

> Why do you think longSummary should take precedence? My thinking was that shorter output is better, as as not to overwhelm the person running it.

My thought was that the short summary output is a subset of the long summary output so when both are requested it made sense to give the full output. It's not a strongly held position :-)

> That argues for pulling summary and longSummary out of that block and putting each of them in their own if statement.
> Thoughts?

That seems fine to me :+1: